docs/configuration/tls.md

```diff
@@ -23,7 +23,9 @@ distinguished_name = req
 subjectAltName = DNS:modelmesh-serving.${NAMESPACE},DNS:localhost,IP:0.0.0.0
 ```
 
-With the generated key/cert, create a kube secret with contents like:
+`${NAMESPACE}` is the namespace where the ModelMesh Serving Service is deployed.
```

Comment:
This could go wrong if the variables are not defined, or not replaced by the user. 

I would explicitly set the variables used in this guide like

> First, define the variables that will be used in the shell scripts below. Feel free to change the values of the variables:

```shell
NAMESPACE="modelmesh-serving"  # the controller namespace, where ModelMesh Serving was deployed
SECRET_NAME="modelmesh-certificate"
```

... and make the code snippets work, like


```shell
openssl req -x509 -newkey rsa:4096 -sha256 -days 3560 -nodes \
    -keyout example.key \
    -out example.crt \
    -subj "/CN=${NAMESPACE}" \
    -extensions san \
    -config openssl-san.config
```

... and:

```
kubectl apply -f - <<EOF
---
apiVersion: v1
kind: Secret
metadata:
  namespace: ${NAMESPACE}
  name: ${SECRET_NAME}
type: kubernetes.io/tls
stringData:
  tls.crt: $(cat example.crt)
  tls.key: $(cat example.key)
  ca.crt: $(cat example.crt)
EOF
```

Or, let `kubectl` create the secret like:

```shell
kubectl create secret generic ${SECRET_NAME} \
        --from-file=tls.key=example.key \
        --from-file=tls.crt=example.crt \
        --dry-run=server \
        -o yaml |
    kubectl -n ${NAMESPACE} apply -f -
```